src/tools/MDBToCBExport.ts

```diff
@@ -0,0 +1,77 @@
+import { CBTools, Type } from "../util/DependencyDownloaderUtils/CBTool";
+import { getActiveConnection, getConnectionId } from "../util/connections";
+import { Constants } from "../util/constants";
+import * as keytar from "keytar";
+import * as vscode from "vscode";
+
+export class MDBToCB {
+    static async export(
+        mDBConnectionString: string,
+        databases: string,
+        collections: string[],
+        indexes: string,
+        cbBucket: string,
+        cbScope: string,
+        context: vscode.ExtensionContext
+    ): Promise<void> {
+        const connection = getActiveConnection();
+        if (!connection) {
+            return;
+        }
+
+        const password = await keytar.getPassword(
+            Constants.extensionID,
+            getConnectionId(connection)
+        );
+        if (!password) {
+            return undefined;
+        }
+        try {
+            // Traverse the collections to build seperate commands
+            for (const collection of collections) {
+                // Build Command
+                const cmd: string[] = [];
+                cmd.push(CBTools.getTool(Type.CB_MIGRATE).path);
+                cmd.push("mongo ");
+                cmd.push("--mongodb-uri");
+                cmd.push(mDBConnectionString);
+                cmd.push("--mongodb-database");
+                cmd.push(databases);
+                cmd.push("--mongodb-collection");
+                cmd.push(collection);
+                cmd.push("--cb-cluster");
+                cmd.push(connection.url);
+                if (indexes) {
+                    cmd.push("--copy-indexes");
+                }
+                cmd.push("--cb-username");
+                cmd.push(connection.username);
+                cmd.push("--cb-password");
+                cmd.push(password);
+                cmd.push("--cb-bucket");
+                cmd.push(cbBucket);
+                cmd.push("--cb-scope");
+                cmd.push(cbScope);
+                // Collection field is optional
```

Comment:
Do we need this?
if not let's remove the comment

Reply:
Yep, we can add this in the next release if CLI supports it by default.